release-notes/9.0/supported-os.md

```diff
@@ -0,0 +1,121 @@
+
+For issues with .NET on operating systems not listed here, please open a GitHub issue in the appropriate .NET repository or contact the OS maintainer community. See [.NET Repos](../../Documentation/core-repos.md) for the repository list.
+
+## Windows
+
+OS                                    | Version                 | Architectures   | Lifecycle
+--------------------------------------|-------------------------|-----------------|----------
+[Windows 10 Client][Windows-client]   | Version 1607+           | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows 11][Windows-client]          | Version 22000+          | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows Server][Windows-Server]      | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Windows Server Core][Windows-Server] | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Nano Server][Nano-Server]            | Version 1809+           | x64             | [Windows Server][Windows-Server-lifecycle]
+
+[Windows-client]: https://www.microsoft.com/windows/
+[Windows-lifecycle]: https://support.microsoft.com/help/13853/windows-lifecycle-fact-sheet
+[win-client-docker]: https://hub.docker.com/_/microsoft-windows
+[Windows-Server-lifecycle]: https://learn.microsoft.com/windows-server/get-started/windows-server-release-info
+[Nano-Server]: https://learn.microsoft.com/windows-server/get-started/getting-started-with-nano-server
+[Windows-Server]: https://learn.microsoft.com/windows-server/
+
+.NET 9 is supported in the x64 emulator on Windows 11 Arm64.
+
+## Linux
+
+OS                                    | Version               | Architectures     | Lifecycle
+--------------------------------------|-----------------------|-------------------|----------
+[Alpine Linux][Alpine]                | 3.17+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
+[Debian][Debian]                      | 11+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
+[Fedora][Fedora]                      | 38+                   | x64               | [Fedora][Fedora-lifecycle]
+[openSUSE][OpenSUSE]                  | 15+                   | x64               | [OpenSUSE][OpenSUSE-lifecycle]
+[Oracle Linux][Oracle-Linux]          | 8+                    | x64               | [Oracle][Oracle-lifecycle]
+[Red Hat Enterprise Linux][RHEL]      | 8+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
+[SUSE Enterprise Linux (SLES)][SLES]  | 12 SP5+               | x64               | [SUSE][SLES-lifecycle]
+[Ubuntu][Ubuntu]                      | 20.04+                | x64, Arm64, Arm32 | [Ubuntu][Ubuntu-lifecycle]
+
+Other distributions are supported at best effort, per [.NET Support and Compatibility for Linux Distributions](../../linux-support.md).
+
+### Libc compatibility
+
+- [glibc][glibc] 2.23 (from Ubuntu 16.04)
```

Comment:
The discussion in https://github.com/dotnet/runtime/issues/91826 suggested to bump this to 18.04. Is it intentional that we are keeping this at 16.04?

cc @richlander

Reply:
You are right. 18.04.

Reply:
```suggestion
- [glibc][glibc] 2.27 (from Ubuntu 18.04)
```